src/_guides/language/spec.md

```diff
@@ -1,73 +1,29 @@
 ---
 layout: default
 title: "Dart Language Specification"
-description: "The Dart language specification and proposed changes."
+description: "The Dart language specification."
 ---
 
-Download the _Dart Programming Language Specification_ from
-the Ecma website:
-
-* <a href="http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-408.pdf"
-   target="_blank">Dart Programming Language Specification, 4<sup>th</sup> Edition</a>
-
+The following information talks about the formal Dart language specification.
 For a gentler introduction to the Dart language, see the
 [Dart language tour](/guides/language/language-tour).
 
-You can find previous editions of the specification at
-[Standard ECMA-408](http://www.ecma-international.org/publications/standards/Ecma-408-arch.htm).
-
-
-## Dart 2 changes
+## Dart 2
 
 Dart 2 has changed the Dart language in many ways,
 some of which are not backward-compatible.
-For information about changes and implementation status,
-see the following resources:
-
-* [Dart 2 Updates](/dart-2)
-* [Dart Language and Library Newsletters](https://github.com/dart-lang/sdk/tree/master/docs/newsletter#dart-language-and-library-newsletters)
-* [Language specification source file (TeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
-* [Informal specifications](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
-
-
-## Changes in the 4<sup>th</sup> edition
-
-The 4<sup>th</sup> edition of the specification has fewer restrictions on mixins.
-The restrictions on mixins up until this edition were:
+For additional details, please see the [Dart 2 changelog](https://github.com/dart-lang/sdk/blob/master/CHANGELOG.md#200---2018-08-07).
 
-* Mixins must inherit directly from Object.
-* Mixins cannot contain super calls.
-* Mixins cannot define constructors.
+The formal specification is still in the process of being completed.
+For information about changes, see the following resources:
 
-Only the last one, mixins cannot define constructors, remains.
+* [Dart language specification source file (TeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
+* [Dart informal specifications](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
 
-## Changes in the 3<sup>rd</sup> edition
+## Dart 1
```

Comment:
Are we referring to Dart 1 as "Dart 1" now? (That would be great.) Otherwise, to be consistent with the rest of the site, it needs to be "Dart 1.x".